src/itwinai/torch/distributed.py

```diff
@@ -56,6 +59,23 @@ def wrapper(self: "TorchDistributedStrategy", *args, **kwargs):
     return wrapper
 
 
+def _initialize_ray() -> None:
```

Comment:
Why the underscore here? I might be remembering this wrong, but I thought that meant that it was a private method belonging to a class. In this case there is no class, so is it still correct to use it here?

Reply:
Good point, I'll address this on the ray-torchtrainer branch though, as this does not really belong here. Same goes for the three comments below, I would keep this PR just about the tutorial if that's okay, these things are all just here because of the merge with the ray-torchtrainer branch (🙄)